src/recordlinker/linking/algorithm_service.py

```diff
@@ -35,7 +35,7 @@ def default_algorithm(session: orm.Session) -> models.Algorithm:
     query = select(models.Algorithm).where(models.Algorithm.is_default == True)  # noqa: E712
     res = session.scalars(query).all()
     if not res:
-        raise ValueError("No default algorithm found in the database.")
+        return None
```

Comment:
Would you mind also adding a test to the test_link_router that shows when we call the endpoints and no default is defined it results in a 422 and not a 500?

Reply:
Added